File: APIs/ChatService/ChatService/DTOs/GroupImageTransport.cs
using Newtonsoft.Json;

namespace ChatService.DTOs
{
    public class GroupImageTransport
    {
        [JsonProperty("groupId")]
        public int GroupId { get; set; }
        [JsonProperty("groupImage")]
        public List<int> GroupImage { get; set; } = null!;
    }
}

File: SocialAppFlutter/social_app/lib/Models/group_image_transport.dart
import 'package:json_annotation/json_annotation.dart';

part 'json_generated_code/group_image_transport.g.dart';

@JsonSerializable(explicitToJson: true)
class GroupImageTransport {
  final int groupId;
  final List<int> groupImage;

  GroupImageTransport(this.groupId, this.groupImage);

  factory GroupImageTransport.fromJson(Map<String, dynamic> json) =>
      _$GroupImageTransportFromJson(json);

  Map<String, dynamic> toJson() => _$GroupImageTransportToJson(this);
}
